src/nvim/buffer.c

```diff
@@ -3281,7 +3281,7 @@ void maketitle(void)
 
   if (p_title) {
     if (p_titlelen > 0) {
-      maxlen = (int)(p_titlelen * Columns / 100);
```

Comment:
as a general note I'd wait to remove casts like this just yet. We might want to support options taking 64-bit values and then these casts will still be needed.

Reply:
I think it'll be easier to revert these if we decide to go with int64. I can't focus on anything with clang-tidy throwing a hissy fit.